draft-ietf-quic-recovery.md

```diff
@@ -659,21 +660,37 @@ treats a Congestion Experienced(CE) codepoint in the IP header as a signal of
 congestion. This document specifies an endpoint's response when its peer
 receives packets with the Congestion Experienced codepoint.
 
+## Initial and Minimum Congestion Window {#initial-cwnd}
+
+QUIC begins every connection in slow start with the congestion window set to
+an initial value.  The RECOMMENDED initial value is the minimum of
+10 * max_datagram_size and max(2 * max_datagram_size, 14720)).  This follows
```

Comment:
Endpoints SHOULD use an initial congestion window of 10 times the maximum datagram size (max_datagram_size), limited to the larger of 14720 or twice the maximum datagram size.